src/Pages/HomePage/HomeComponents/NewArrivalCard.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react'
 import '../HomePage.css'
+import { Link } from 'react-router-dom'
 
-const NewArrivalCard = ({ imgSrc, title }) => {
+const NewArrivalCard = ({ imgSrc, title, filterGameKey, filterCatKey }) => {
     return (
         <div className="img-wrapper pos-rel">
             <div className="grid-txt-overlay pos-abs">
                 <p className="rg-title light-txt center-txt">{title}</p>
-                <a href="./Products/products.html" >
+                <Link to="/shop" state={{ gameEnabled: filterGameKey, filterEnabled: filterCatKey }}>
                     <button className="btn subtle-btn mg-t-10 center-items">Explore <i className="fa fa-solid fa-arrow-right"></i>
                     </button>
-                </a>
+                </Link>
             </div>
             <img className="img-resp" src={imgSrc} alt="Xiao Genshin Poster" />
```

Comment:
keep alt value Dynamic!

Reply:
oh thanks for pointing it out. completely overlooked it.